docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+
+---
+title: Configuration
+---
+
+vSphere Docker Volume Service consists of two components the VIB and Docker plugin. The Docker plugin requires configuration which is described in this section
+
+The Docker volume plugin at the moment supports vSphere and Photon platforms. All the volume provisioning and management operations are supported on both the platforms. Both the platforms can be used to provision VMDK based volumes and can be attached to containers. 
+
+The configurations used by a driver while performing operations are read from a JSON file and the default location where it looks for it /etc/docker-volume-vsphere.conf. You can also override it to use  a different configuration file by providing --config option and the full path to the file. Finally the parameters passed on the CLI override the one from the configuration file.
+
+## Docker Volume Driver
+
+The vSphere volume driver can be used on a standalone or cluster of ESXi servers via the ESXi service. ESXi service needs to be installed and running on all servers. The volumes are created on ESXi host using the VIM (Virtual Infrastructure Management) APIs.
```

Comment:
Please reword, this is confusing. The vSphere volume driver isn't using on ESXi servers, its used inside VMs running on ESXi servers.

Do we want to call it the ESXi service or something like "ESXi Container Volume Service"? ESXi service seems unclear exactly what its meant for.

Para could be reworded as "The vSphere volume driver is run as a managed Docker volume plugin on any Linux VM that is hosted on standalone or clustered ESXi servers and has Docker version 17.03 or later installed in it". The volume driver requires the ESXi container volume service to be installed on the ESXi host to be able to create storage volumes.

I'm not sure why VIM needs to be mentioned here at all.